jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/newcase/NewCaseInstanceViewImpl.java

```diff
@@ -99,21 +118,35 @@ public void cleanForm() {
         caseTemplatesList.setValue("");
         caseTemplatesList.getElement().focus();
         caseTemplatesList.enable();
+        ownerNameInput.setValue("");
         clearErrorMessages();
     }
 
     private boolean validateForm() {
         clearErrorMessages();
 
+        boolean valid = true;
+
         if (isNullOrEmpty(caseTemplatesList.getValue())) {
             caseTemplatesList.getElement().focus();
             definitionNameHelp.setTextContent(translationService.format(PLEASE_SELECT_CASE_DEFINITION));
             caseDefinitionNameGroup.setValidationState(ValidationState.ERROR);
-            return false;
-        } else {
+            valid = false;
+        }
+
+        if (isNullOrEmpty(ownerNameInput.getValue())) {
+            ownerNameInput.focus();
+            ownerNameHelp.setTextContent(translationService.format(PLEASE_PROVIDE_CASE_OWNER));
+            ownerNameGroup.setValidationState(ValidationState.ERROR);
+            valid = false;
+        }
```

Comment:
Form user experience point of view, maybe making case owner field mandatory for every case instance is too strong requirement.
My understanding is that case owners in most of the cases will be case workers, who will also usually start a case instance. So I think we should allow the flexibility of skipping owner field for those cases and making the user who stared the case instance the owner by default.